lib/processor.js

```diff
@@ -36,30 +36,53 @@ exports.work = (audioEvent) => {
   });
 }
 
-// upload to final s3 destination
-exports.upload = (localPath, filePath, fileName) => {
-  let localStream = fs.createReadStream(localPath);
-  let fileParts = path.parse(fileName);
+// upload mp3 and make copies as needed
+exports.uploadAndCopy = (file) => {
+  let dest = `${file.path}/${file.name}`;
+  let type = file.contentType;
+  if (!type && file.format) {
+     type = `audio/${file.format}`;
```

Comment:
If ffprobe detects the format, we can pretty safely guess the mime-type.  (Format will be "mp2" / "mp3" / "ogg" / etc).